src/ol/proj/epsg3857.js

```diff
@@ -31,6 +31,13 @@ export const EXTENT = [-HALF_SIZE, -HALF_SIZE, HALF_SIZE, HALF_SIZE];
  */
 export const WORLD_EXTENT = [-180, -85, 180, 85];
 
+/**
+ * Maximum safe value in y direction
+ * @const
+ * @type {number}
+ */
+export const MAX_SAFE_Y = RADIUS * Math.log(Math.tan((Math.PI * 180) / 360));
```

Comment:
This looks a bit weird calculating with radians.
```suggestion
export const MAX_SAFE_Y = RADIUS * Math.log(Math.tan(Math.PI / 2));
```